stdlib/public/Concurrency/Task.swift

```diff
@@ -522,12 +520,18 @@ extension Task {
 @available(macOS 9999, iOS 9999, watchOS 9999, tvOS 9999, *)
 extension Task {
 
-  /// Explicitly suspend the current task, potentially giving up execution actor
-  /// of current actor/task, allowing other tasks to execute.
+  /// Suspends the current task and allows other tasks to execute.
   ///
-  /// This is not a perfect cure for starvation;
-  /// if the task is the highest-priority task in the system, it might go
-  /// immediately back to executing.
+  /// A task can voluntarily suspend itself
+  /// in the middle of a long-running operation
+  /// that doesn't contain any suspension points,
+  /// to let other tasks run for a while
+  /// before execution returns back to this task.
+  ///
+  /// Note that if this task is the highest-priority task in the system,
+  /// the executor immediately resumes execution of the same task.
+  /// As such,
+  /// this method isn't necessarily a way to avoid resource starvation.
```

Comment:
(a bit weird formatting in this comment?)

Reply:
What looks weird to you?  If it's just the placement of line breaks, that's a convention we've used elsewhere in documentation to reduce noise in diffs.  Instead of breaking at the 80 column mark and rewrapping, we break lines at clause and sentence boundaries, so they don't need to be rewrapped after revising.

Reply:
I see, that's a new technique to me but sounds fine I guess , thanks for explaining